packages/walletconnect/src/index.ts

```diff
@@ -129,7 +130,7 @@ function walletConnect(options?: WalletConnectOptions): WalletInit {
                   // Check if connection is already established
                   if (!this.connector.connected) {
                     // create new session
-                    this.connector.createSession().then(() => {
+                    this.connector.createSession(connectFirstChainId ? {chainId: parseInt(chains[0].id, 16)} : undefined).then(() => {
```

Comment:
@mahmud-bn Just double checking here. Do we need to add some check to make sure that `chains` and `chains[0]` are defined? 
For instance:
```suggestion
                    this.connector.createSession(connectFirstChainId && chains && chains[0] ? {chainId: parseInt(chains[0].id, 16)} : undefined).then(() => {
```

Reply:
@mahmud-bn Awesome! Just wanted to double check